zap/zap.gradle.kts

```diff
@@ -163,6 +164,7 @@ val japicmp by tasks.registering(JapicmpTask::class) {
         reportName = "japi.html"
         isAddDefaultRules = true
         addRule(JApiChangeStatus.MODIFIED, AcceptMethodAbstractNowDefaultRule::class.java)
+        addRule(JApiChangeStatus.MODIFIED, AcceptMethodNewDefaultRule::class.java)
```

Comment:
Please, suppress the incompatibility instead.

Reply:
didn't get you, can you please elaborate?

Reply:
got it, please review now.